substrate/frame/broker/src/dispatchable_impls.rs

```diff
@@ -105,7 +105,15 @@ impl<T: Config> Pallet<T> {
 			cores_offered: 0,
 			cores_sold: 0,
```

Comment:
We would not need to include `sale_index` in the event emitted when sale starts?

Runtime devs and users should query the `SaleInfo` for the current sale Id?

Reply:
Is there a way to know how many sales have elapsed since? this is for migration purposes.

Reply:
On Kusama that might be fiddly because we re-started sales (we have a "sale" which is one week long) - raises the question: do we count that as sale 0 or just ignore it as the symptom of a bug? Would be interesting to see how others have labelled it. I guess you could just have an input `sales_at_block` along with the start block and current block or whatever. I'm sure you could simplify that too, but that's the fully general one for which the upgrade wouldn't need to be timed.

Reply:
> do we count that as sale 0 or just ignore it as the symptom of a bug?

We just ignore. By restarting we have voided any sales etc any way? (don't remember the exact details)

Reply:
> > do we count that as sale 0 or just ignore it as the symptom of a bug?
> 
> We just ignore. By restarting we have voided any sales etc any way? (don't remember the exact details)

The start block has to be specific to the relaychain(i.e Wested, Kusama and Polkadot)?